.github/workflows/gql_deprecated_fields_report.yml

```diff
@@ -0,0 +1,69 @@
+# This workflow will build a report with the used deprecated GQL fields
+# Parent workflow must pass the secrets: inherit option
+# See example: on-pr-example.yml
+name: Deprecated fields report
+
+on:
+  workflow_call:
+    inputs:
+      schema_urls:
+        description: "Schema URLs to lint, comma separated. E.g: https://one.com/api/graphql,https://two.com/clients/graphql"
+        required: true
+        type: string
+        default: ""
+      query_files:
+        description: "Paths of query files to lint, comma separated. E.g: queries/**/*.graphql,api/**/*.gql"
+        required: true
+        type: string
+        default: ""
+      ignore_files:
+        description: "Paths of files to ignore, comma separated. E.g: .graphql/**/local.*.gql,other/path/**/*.graphql"
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  reportUsedDeprecatedFields:
+    name: Report used deprecated fields
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Install gql-lint
+      - name: Install gql-lint
+        run: curl https://raw.githubusercontent.com/sketch-hq/gql-lint/main/install.sh | /bin/bash -s -- latest
+        shell: bash
+
+      # Install jo
+      - name: Install jo
+        run: sudo apt-get install jo
+        shell: bash
+
+      - name: Find all deprecated fields
+        shell: bash
+        id: report
+        run: |
+          gql-lint deprecation \
+          --schema "${{ inputs.schema_urls }}" \
+          --ignore "${{ inputs.ignore_files }}" \
+          "${{ inputs.query_files }}" \
```

Comment:
Does this really work well with the commas? globbing works as expected? And what if there are spaces on the paths? Maybe we solved this in the tool itself but it was an issue for me in the other workflow

Reply:
do you mean for the schema URLs? it did work well on this run: https://sketch.slack.com/archives/C04PRCA120N/p1677159279256719?thread_ts=1677159279.070249&cid=C04PRCA120N

Reply:
No, the input queries, when they are coma separated and path with spaces, i.e., the real case we have now for Sketch app. I think it works fine with FE because it is one only element (no commas) and has no spaces in the path

Reply:
it works with a list of paths, they [aren't comma separated](https://github.com/sketch-hq/Sketch/blob/develop/.github/workflows/deprecated_fields_report.yml#L34-L35) now and they won't be when they're passed here. Not sure if I'm missing something?

Reply:
as discussed in slack, it's not really necessary to support commas and this works fine just as it is